### ISSUE TITLE
zdiff3 stops sheldon lock installing plugins

### ISSUE THREAD

**Issue by jonnybarnes** [rossmacarthur/sheldon#146]
When running `sheldon lock` to install the defined plugins I get the following error:

```sh
$ sheldon lock
Loaded ~/.sheldon/plugins.toml

error: failed to install source `https://github.com/romkatv/zsh-defer`
  due to: failed to set HEAD to `57a6650ff262f577278275ddf11139673e01e471`
  due to: unknown style 'zdiff3' given for 'merge.conflictstyle'; class=Checkout (20)
```

**Comment by rossmacarthur** [comment]
Hi @jonnybarnes, thanks for the report. I am able to replicate. It looks like this is due to the the fact that we vendor libgit2 in sheldon and the currently vendored version is older than the 'zdiff3' feature in libgit2. This will be fixed in the next release of sheldon.

**Comment by rossmacarthur** [comment]
Resolved in 3a88ce7e314e724987657f3e7ed2934820412e1d, with the libgit2 upgrade. This is released in version 0.7.0.
